README.md

```diff
@@ -34,8 +34,10 @@ const runner = new Runner(test-files, ignored-files, params);
 
 runner.run()
     .then(results => {
-        // Optional custom reporting using the raw data
+        // Optional custom reporting using the raw data for a full report
         const report = runner.generateReport(params.title, results);
+        // Option custom reporting using the raw data for a minimal report (only shows slows and failures)
+        const report = runner.generateMinimalReport(params.title, results);
```

Comment:
Probably shouldn't show two `const`s with the same name, even if it is just in a readme. Also, the comment should use 'Optional'